<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_gestaoVendas</class>
 <widget class="QDialog" name="fm_gestaoVendas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>420</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Gerar Relat√≥rio PDF</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_listarVenda">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>661</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_listarProdutosVendas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>661</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Data inicial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="de_dataInicial"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Data final</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="de_dataFinal"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_filtrarData">
      <property name="text">
       <string>Filtrar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Mostrar todas vendas</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
